crates/topos-clock/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+//! This crate is responsible for managing the clock pace
```

Comment:
The PR description is a bit terse, but it sounds like this might be a temporary solution. I still think we should have a more complete top-level justification for what this crate is doing. Especially important imo is to indicate to readers if this code is a stop-gap solution until we have something better in place (if that is indeed the case).